.github/workflows/example.yml

```
@@ -41,13 +48,13 @@ jobs:
         uses: ./
         with:
           working-directory: example
-          stack-arguments: --resolver ${{ matrix.resolver }}
-          cache-prefix: ${{ matrix.resolver }}/
+          stack-arguments: --resolver ${{ matrix.stack.resolver }}
+          cache-prefix: ${{ matrix.stack.resolver }}/
 
-      - if: ${{ runner.os != 'windows' }}
+      - shell: bash
         run: |
-          [[ "${{ steps.stack.outputs.compiler }}" = "ghc-8.10.4" ]]
-          [[ "${{ steps.stack.outputs.compiler-version }}" = "8.10.4" ]]
+          [[ "${{ steps.stack.outputs.compiler }}" = ${{ format('ghc-{0}', matrix.stack.ghc) }} ]]
```

Can you just do this?

```suggestion
          [[ "${{ steps.stack.outputs.compiler }}" = ghc-${{ matrix.stack.ghc }} ]]
```

Seems simpler.

---

applied the fix in https://github.com/freckle/stack-action/pull/35/commits/65802cae1131158eb58f11f7825e0378cc1f489e